Assets/Src/Scripts/UI/HeaderButtonEvents.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021-2022 The SeedV Lab.
```

Comment:
Overall style issues:

- Google style: "Indentation of 2 spaces, no tabs." You are using 4-space indentations.
- There are trailing spaces at line 24, line 33, line 38, and line 71. Please delete those trailing spaces.